src/test/resources/Logout.feature

```diff
@@ -0,0 +1,12 @@
+Feature: MakeAppointment
+
+  An a user
+  I want to sign in
+  So I can use the app
+
+  @automated
+  Scenario: User can logout at make appointment screen
+    Given user signed in with "John Doe" and "ThisIsNotAPassword"
```

Comment:
thay tất cả các step `" Given user signed in with "John Doe" and "ThisIsNotAPassword" `bằng step:  `Given user signed in with acc "John Doe" and "ThisIsNotAPassword"`, có step với hành động như thế rồi còn thêm mới làm gì